src/System.ComponentModel.TypeConverter/src/System/Drawing/ColorConverter.cs

```diff
@@ -53,7 +53,7 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
             string strValue = value as string;
             if (strValue != null)
             {
-                return ColorConverterCommon.ConvertFromString(strValue, culture);
+                return ColorConverterCommon.ConvertFromString(strValue, culture ?? CultureInfo.CurrentCulture);
```

Comment:
FWIW, the value is defaulted in that method: https://github.com/dotnet/corefx/blob/master/src/Common/src/System/Drawing/ColorConverterCommon.cs#L34